README.md

```diff
@@ -1,7 +1,36 @@
-# falcon-helm
+# CrowdStrike Falcon Helm Chart and Helm Operator
+
+[Falcon](https://www.crowdstrike.com/) is the [CrowdStrike](https://www.crowdstrike.com/)
+platform purpose-built to stop breaches via a unified set of cloud-delivered
+technologies that prevent all types of attacks — including malware and much
+more.
+
+The CrowdStrike Falcon Helm Chart and the Helm Operator are designed to deploy
+and manage the Falcon sensor on your Kubernetes cluster of choice.
+[Helm Charts](https://helm.sh/docs/topics/charts/Helm) and Helm-based Kubernetes
+Operators are packaging methodologies for Kubernetes. Which methodology you use
+is dependent on what the Kubernetes vendor and implementation supports.
```

Comment:
```suggestion
The CrowdStrike Falcon Helm Chart and the Helm Operator are designed to deploy
and manage the Falcon sensor on your Kubernetes cluster of choice.

The Falcon platform supports Open
Container Initiative (OCI)-based containers such as Docker and Kubernetes and
also self-managed and hosted orchestration platforms such as GKE (Google
Kubernetes Engine), EKS (Amazon Elastic Kubernetes Service), ECS (Amazon
Elastic Container Service), AKS (Azure Kubernetes Service) and Red Hat's OpenShift.
```

Reply:
Taken from bottom of page 4 https://www.crowdstrike.com/wp-content/uploads/2020/04/container-security-solution-brief.pdf